.github/workflows/notify-module-version.yml

```diff
@@ -0,0 +1,28 @@
+name: Notify MODULE_VERSION changed
+
+on:
+  push:
+    branches:
+    - master
+    - release-*
+    paths:
+    - kernel-modules/MODULE_VERSION
+
+jobs:
+  notify:
+    runs-on: ubuntu-22.04
+    steps:
+    - name: Slack notification
+      uses: rtCamp/action-slack-notify@v2
```

Comment:
Release improvers folks use different action. See here
https://github.com/stackrox/stackrox/blob/8124e3889295a152f6a76c979df03b516c1e3060/.github/workflows/finish-release.yml#L120-L137

Maybe they also can help with the token?

Reply:
I'll ask them. That action looks very flexible but harder to use, since it needs to have the payload manually created and this one you just set a few env variables and are on your way.